pkg/cli/admin/upgrade/recommend/examples/4.12.16-longest-not-recommended.output

```diff
@@ -1,23 +1,22 @@
-Upgradeable=False
-
-  Reason: AdminAckRequired
-  Message: Kubernetes 1.26 and therefore OpenShift 4.13 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6958394 for details and instructions.
-
 Upstream: https://api.integration.openshift.com/api/upgrades_info/graph
 Channel: stable-4.13 (available channels: candidate-4.12, candidate-4.13, eus-4.12, eus-4.14, fast-4.12, fast-4.13, stable-4.12, stable-4.13)
 
 Updates to 4.13:
 
   Version: 4.13.50
   Image: quay.io/openshift-release-dev/ocp-release@sha256:6afb11e1cac46fd26476ca134072937115256b9c6360f7a1cd1812992c065f02
-  Reason: EvaluationFailed
-  Message: Exposure to ARODNSWrongBootSequence is unknown due to an evaluation failure: client-side throttling: only 8m30.897651224s has elapsed since the last match call completed for this cluster condition backend; this cached cluster condition request has been queued for later execution
+  Reason: MultipleReasons
+  Message: Kubernetes 1.26 and therefore OpenShift 4.13 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6958394 for details and instructions.
+  
+  Exposure to ARODNSWrongBootSequence is unknown due to an evaluation failure: client-side throttling: only 8m30.897651224s has elapsed since the last match call completed for this cluster condition backend; this cached cluster condition request has been queued for later execution
```

Comment:
Probably not something to be addressed in this PR, just a thought. If we're going to rely on conditional risks more we'll definitely need to improve the UX for `MultipleReasons`, the individual risks get really lost in the `Message` concatenation. How many risks are there? What names do they have?

Reply:
one issue with trying to make `reason` a clear summary on its own is whether we want to distinguish "risk matches" vs. this^ kind of "risk matching unknown".  I'm not too worried, because the entry you comment on here has a `message` covering the distinction, and I want customers to wade through that whole `message`.  I'm not expecting the tabular output to be actionable without using `--version ...` or some other step to get more details on a 4.y.z that you're thinking about from the table output.  Folks might be tempted to go deep in history to pick out a release that isn't exposed to some risk that modern releases aren't exposed to, but that's going to leave the cluster exposed to all sorts of bugs since fixed in modern releases, and I dunno if we want to make it especially convenient.  That said, any update is going to fix bugs, so if you go from 4.y.ancient to 4.y.moderatelyOld, that's still better than being spooked by 4.y.new regressions and staying on 4.y.ancient.